modules/openapi-generator/src/main/resources/go/model.mustache

```diff
@@ -35,9 +35,95 @@ type {{classname}} struct {
 {{#description}}
 	// {{{description}}}
 {{/description}}
-	{{name}} {{#isNullable}}*{{/isNullable}}{{{dataType}}} `json:"{{baseName}}{{^required}},omitempty{{/required}}"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
+	{{name}} *{{{dataType}}} `json:"{{baseName}},omitempty"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
```

Comment:
also, it's breaking backward-compatibility. the field shouldn't twiddle between pointer and non-pointer at least

Reply:
Yeah, I've had a discussion about this with @wing328 and we'll probably create another go client generator to accept this change (and possibly other breaking changes) that would otherwise be unacceptable for 4.1.x and would have to wait for 5.0.0 - which would be a loooong time.

Reply:
@bkabrda For the new go generator, would you mind tagging it with "Experimental" 

https://github.com/OpenAPITools/openapi-generator/blob/6e1c8976fcf0d42904240ff7218f4105b03121f8/modules/openapi-generator-core/src/main/java/org/openapitools/codegen/meta/Stability.java#L19

It would look like this in the new generator's constructor:

```
        generatorMetadata = GeneratorMetadata.newBuilder(generatorMetadata)
                .stability(Stability.EXPERIMENTAL)
               .build();
```

This will allow us to tag/filter generators to clearly present which ones are considered stable to users.